react/error.tsx

```diff
@@ -21,22 +21,53 @@ class ErrorPage extends Component {
       this.setState({ enabled: true })
     }, 5000)
 
-    if (!isAdmin()) {
-      return
+    try {
+      const error = window?.__ERROR__
+      const requestId = window?.__REQUEST_ID__
+      const defaultError =
+        'Render Runtime renderered an error page and there is no error or request id available'
+
+      const runtime = window?.global?.__RUNTIME__ ?? {}
+      const errorInfo = this.extractErrorInfo(runtime)
+
+      if (!isAdmin() || errorInfo.admin_production === false) {
+        return
+      }
+
+      getCurrentScope().setTags(errorInfo)
+
+      if (error) {
+        captureException(error)
+      } else if (requestId) {
+        captureException(requestId)
+      } else {
+        captureException(defaultError)
+      }
+    } catch (e) {
+      captureException(e)
     }
+  }
 
-    const error = window?.__ERROR__
-    const requestId = window?.__REQUEST_ID__
-    const defaultError =
-      'Render Runtime renderered an error page and there is no error or request id available'
-
-    if (error) {
-      captureException(error)
-    } else if (requestId) {
-      captureException(requestId)
-    } else {
-      captureException(defaultError)
+  private extractErrorInfo(runtime: any): any {
+    const {
+      account = null,
+      workspace = null,
+      culture: { locale } = { locale: null },
+      route: { path } = { path: null },
+      loadedDevices = [null],
+      production = null,
+    } = runtime
+
+    const errorInfo = {
+      admin_account: account,
+      admin_workspace: workspace,
+      admin_locale: locale,
+      admin_path: path,
+      admin_device: loadedDevices[0],
+      admin_production: production,
```

Comment:
It occurs to me that we are better off with regular `account`, `workspace`, etc nomenclature as it's the one used across other o11y services

```suggestion
      account,
      workspace,
      locale,
      path,
      device: loadedDevices[0],
      production,
```

Reply:
I've changed that because of Sentry's documentation
> Some tags are automatically set by Sentry. We strongly recommend against overwriting these [tags](https://docs.sentry.io/concepts/search/searchable-properties/#search-properties). Instead, name your tags with your organization's nomenclature. If you overwrite an automatically set tag, you must use [explicit tag syntax](https://docs.sentry.io/concepts/search/#explicit-tag-syntax) to search for it.

Sentry already has `device` and `production` tags for example.
[Sentry tags documentation](https://docs.sentry.io/platforms/javascript/enriching-events/tags/)

Reply:
Nice, good catch!!!